src/reducers/location.js

```diff
@@ -0,0 +1,10 @@
+import { LOCATION_QUERY } from "../actions/location";
+
+export default (state = [], action = {}) => {
+  switch (action.type) {
+    case LOCATION_QUERY:
+      return action.payload;
+    default:
+      return state;
+  }
+};
```

Comment:
This reducers seem simple enough to join then together maybe, think about it. You can have one `event` reducer with a state like
```
const initialState = { categories: [], selectedEvent: null | {}, byLocation: [] }
```

Reply:
if you see the data growing in the future then maybe keep them apart or just join `event` and `location`